kubeflow/README.md

```diff
@@ -62,7 +62,6 @@ Provide actual value for the following variables in `env.sh`:
 ```
 KF_NAME=<kubeflow-cluster-name>
 KF_PROJECT=<gcp-project-id>
-KF_DIR=<current-kubeflow-directory-path>
```

Comment:
KF_DIR is a bash only env var, so I think we can only keep it in documentation. It's kind of problematic in env.sh, because we might want to run env.sh from different working directories.

Reply:
I see, then we should probably extract this line in documentation, and ask user to run manually.